src/init/consented/fill-slot-listener.ts

```diff
@@ -27,20 +27,28 @@ const isCustomEvent = (event: Event): event is CustomEvent => {
  * These events will not be received from a restricted iframe such, such as a
  * cross-origin or safeframe iframe.
  */
-export const createSlotFillListener = () => {
+const createSlotFillListener = () => {
 	document.addEventListener('gu.commercial.slot.fill', (event: Event) => {
-		if (isCustomEvent(event)) {
-			const { slotId, additionalSizes } = (<ExternalSlotCustomEvent>event)
-				.detail;
+		window.googletag.cmd.push(() => {
```

Comment:
What if something went wrong with googletag what will happen then? I understand that the slot won't be filled but what we will have instead?